Makefile

```diff
@@ -658,6 +660,11 @@ test_versionable: install_rs_build_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) test --profile $(CARGO_PROFILE) \
 		-p tfhe-versionable
 
+.PHONY: test_backward_compatibility
+test_backward_compatibility: install_rs_build_toolchain
```

Comment:
make the target clone data automatically ?

and if needed we can have a ci target that does not clone automatically if it conflicts with the actions checkout

Reply:
fixed